src/calendar/Calendar.ts

```diff
@@ -375,13 +374,11 @@ export class CalendarBase<P extends CalendarProperties = CalendarProperties> ext
 	}
 
 	protected renderPagingButtonContent(type: Paging, labels: CalendarMessages): DNode[] {
-		const iconClass = type === Paging.next ? iconCss.rightIcon : iconCss.leftIcon;
+		const iconType = type === Paging.next ? 'rightIcon' : 'leftIcon';
```

Comment:
could this be `IconType.rightIcon` ? Might be better using the exported type. There's a few instances like this within this PR.

Reply:
Right now `IconType` is just a type, so there is no `IconType.rightIcon`.